python/tournament-scheduling/tests/test_constraints.py

```diff
@@ -0,0 +1,68 @@
+from timefold.solver.test import ConstraintVerifier
+from tournament_scheduling.domain import *
+from tournament_scheduling.constraints import *
+
+constraint_verifier = ConstraintVerifier.build(define_constraints, TournamentSchedule, TeamAssignment)
+# Sample teams and days (equivalent to static variables in Java)
```

Comment:
I don't think the latter part of the comment is helpful. It would only confuse a Python developer looking at it.
I would remove this comment, since it is obvious this is sample data (and there no such comment in Java).
```suggestion
```